manifests/image-references

```diff
@@ -38,3 +38,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-vsphere-cloud-controller-manager
+  - name: powervs-cloud-controller-manager
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-powervs-cloud-controller-manager
```

Comment:
Is this image in payload yet?

Reply:
not yet, lets wait
/hold

Reply:
The [ART ticket](https://issues.redhat.com/browse/ART-3885) is closed and image will get into release image post this image-reference addition I feel.

The sample image looks like:
```
registry-proxy.engineering.redhat.com/rh-osbs/openshift4-ose-powervs-cloud-controller-manager:v4.11.0-202203221643.p0.ga6bfa07.assembly.stream 
```